package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "teams-for-linux",
-  "version": "1.0.49",
+  "version": "1.0.50",
```

another release has increased this to 50, so adding another number :D
```suggestion
  "version": "1.0.51",
```